projects/control-service/projects/helm_charts/pipelines-control-service/templates/ingress.yaml

```diff
@@ -16,7 +16,14 @@ spec:
     http:
       paths:
       - path: {{ .Values.ingress.path | default "/" }}
-        pathType: {{ .Values.ingress.pathType | default "ImplementationSpecific" }}
+        pathType: "ImplementationSpecific"
+        backend:
+          service:
+            name: {{ .Release.Name }}-ui
+            port:
+              number: 8091
+      - path: {{ .Values.ingress.path | default "/data-jobs" }}
```

Comment:
So what's the full path to control service going to be exactly? I'd just like to avoid a situation where the url is `{host}:{port}/data-jobs/data-jobs`

Reply:
Yeah it will just be a single /data-jobs.